### ISSUE TITLE
Program option doesnt work

### ISSUE THREAD

**Issue by MilanObrtlik** [paulrosen/abcjs#111]
```javascript
new ABCJS.Editor("abc", {
    paper_id: "paper0",
    generate_midi: true,
    midi_id: "midi",
    generate_warnings: true,
    warnings_id:"warnings",
    midi_options: {
        program: 40
    },
    render_options: {
        listener: { highlight: selectionCallback }
    }
}
```
And acoustic grand piano is still playing.

**Comment by paulrosen** [comment]
Yes, I was having a problem with performance when changing instruments, so I don't support that yet. I'll try to improve the midi processing in the future. Sorry.

**Comment by MilanObrtlik** [comment]
Yes please, changing instruments is very cool feature. Sadly create fix for this feature is over my skills in JS.

**Comment by paulrosen** [comment]
This will be in the next release -- soon, I hope.

**Comment by paulrosen** [comment]
Will be fixed in v3.3.0, out soon.
